content/2_Bronze/Time_Comp.mdx

```diff
@@ -451,3 +452,74 @@ triplets.
 <!-- Bitsets are significantly faster (and so are bitwise operations vs iterating through bits). I don't want to make it sound insignificant -->
 
 For now, don't worry about optimizing constant factors -- just be aware of them.
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    What's time complexity?
+    <Quiz.Answer>
+      Measuring the amount of memory an algorithm consumes.
+      <Quiz.Explanation>
+        This is known as space complexity.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Measuring the number of operations an algorithm performs.
+      <Quiz.Explanation>
+        You'd always want your algorithm to run as fast as possible!
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The time it takes you to solve a problem.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What time complexity would a mathematical formula take?
+    <Quiz.Answer correct>
+      $\mathcal{O}(1)$
+      <Quiz.Explanation>
+        No matter how big a certain number is, a mathematical formula would execute the same number of operations.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+      <Quiz.Explanation>
+        Would this algorithm take more steps because of a larger number?
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N^2)$
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What does Big O notation express?
+    <Quiz.Answer>
+      The best case scenario.
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      The worst case scenario.
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The average scenario.
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What's the time complexity of a for loop?

```

Comment:
change to "a for loop from 1 to N" or say "What's the complexity of a for loop through N elements," its not clear what N is here